### PR TITLE
Update rehooks extension

### PR DESCRIPTION
**Pull request by Pyr33x**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Pyr33x** [comment]
I guess this PR now has the potential to be merged.

File: extensions/rehooks/package.json [hunk 1]
```diff
@@ -11,10 +11,17 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "search-hooks",
```
Comment (pernielsentikaer) [inline extensions/rehooks/package.json:14]: Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂

Comment (Pyr33x) [reply extensions/rehooks/package.json:14]: I don't have access to my laptop rn, but looks like you assigned this to yourself

**Comment by pernielsentikaer** [comment]
I'm just the assigned reviewer on this PR 🙂

When can you do it yourself?

**Comment by Pyr33x** [comment]
> I'm just the assigned reviewer on this PR 🙂
> 
> When can you do it yourself?

It's okay, I'm on the way to do it right now.

**Comment by Pyr33x** [comment]
It's finally here 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by Pyr33x** [comment]
> Hi 👋 
> 
> Looks good to me, approved 🔥

Thank you sir ♥️🫡

**Pull request merged**